sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3261,12 +3294,14 @@ SUBROUTINE MISCLN
                  DO I=1,IM
                    IF(T1D(I,J) < spval) THEN
                       GRID1(I,J) = EGRID1(I,J)
-                      IF (SUBMODELNAME == 'RTMA') MUCAPE(I,J) = GRID1(I,J)
+                      IF (SUBMODELNAME == 'RTMA' .OR. MODELNAME == 'FV3R') THEN
```

Comment:
May I ask the reason of adding model name "FV3R"?

Reply:
This additional conditional model check (FV3R) was added in order to ensure that the field would be accessible for the calculation of the SPC effective layer supercell parameter which needs to be an output product for the RRFS-based 3D-RTMA workflow.